tests/integration/README.md

```diff
@@ -0,0 +1,47 @@
+# Kiali Integration Tests
+
+### Project Structure
+
+* `tests/integration/`:  Root directory
+    *  `tests`:  Tests
+    *  `utils`:  Support utilities
+
+
+## Environment Setup
+
+* System dependencies that will need to be available
+    *  `oc`
+    *  `go`
+    *  `make`
+    *  `npm`
+    *  `yarn`
+
+* It is expected that the following have already been deployed in Openshift
```

Comment:
Is there a reason this README is geared towards OpenShift? I do not see instructions here on how to run these integration tests on non-OpenShift environments (like minikube). We should have the ability to run these i-tests on an OpenShift and a non-OpenShift cluster - instructions on how to do both are needed.

Reply:
so far oc is not required, will remove it for now